src/tools/cephfs_mirror/Mirror.cc

```diff
@@ -531,13 +536,6 @@ void Mirror::update_fs_mirrors() {
         ctx->complete(0);
       }
     }
-
-    if (check_blocklist) {
-      m_last_blocklist_check = now;
-    }
-    if (check_failure) {
-      m_last_failure_check = now;
-    }
```

Comment:
ugh; nice catch! this should make the failed replayers restart correctly and might fix an edge case like https://tracker.ceph.com/issues/62344 i guess?

Reply:
Not really -- its just wont restart the blocklisted or failed instance as it should (on timeout expiry).